src/js/data/names.js

```diff
@@ -3436,7 +3436,7 @@ let first = {
         ["A.C.", 5],
         ["A.J.", 63],
         ["A.W.", 64],
-        ["ATorri", 65],
```

Comment:
I've reverted it, but I am also going to re-open it as an issue for `tools/names.js`. If you could share with me just the snippet of the Draft Express data that includes http://basketball.realgm.com/player/ATorri-Shine/Summary/60489 then I'd be happy to try and debug it. I think there's a line between "the name is misspelled because the parents intentionally misspelled it" (a la 'Dwyane Wade') and "the parser read the name wrong". We know that these names are supposed to have an apostrophe. 

N'Doye is another example: https://en.wikipedia.org/wiki/Dame_N%27Doye